src/routes/+page.svelte

```diff
@@ -45,16 +47,19 @@
 	</section>
 	
 	<!-- Hero -->
-	<section id="hero" class={`py-8  bg-[size:200%] md:bg-cover min-h-[calc(100vh-${navSize}px)]`}>
-		<div class="flex flex-col items-center gap-4 px-[10%]">
-			<Text type="callout" color="darkPrimary" class="rounded-full px-4 sm:px-5 py-2 bg-primary-dark">Backed by top investors</Text>
-			<Text type="hero" color="darkPrimary" class="sm:max-w-[60%] text-center">Turn all Partnerships into Revenue</Text>
+	<section id="hero" class="flex flex-col gap-6 sm:gap-8 pt-8 pb-32 sm:pb-24">
+		<div class="flex flex-col items-center mx-4 sm:mx-28">
+			<Toast type="hero" textColor="hero">Backed by top investors</Toast>
+			<Text type="hero" color="darkPrimary" class="pt-5 pb-4 sm:max-w-[60%] text-center">Turn all Partnerships into Revenue</Text>
 			<Text type="subhead" color="darkSecondary" class="sm:max-w-[55%] text-center">
 				Our AI Partnerships Intelligence Platform analyzes sales calls to uncover partner
 				opportunities. Our insights and workflows empower you to boost revenue and close deals faster
 				through partnerships.
 			</Text>
-			<Button type="primary" href={getEarlyAccessHref}>{getEarlyAccessText}</Button>
+		</div>
+		<div class="flex flex-col relative">
+			<img class="hidden sm:block sm:-mt-6" src={HeroBackground} alt="Background logo" />
+			<Button class="place-self-center sm:z-10 sm:absolute" type="primary" href={getEarlyAccessHref}>{getEarlyAccessText}</Button>
```

Comment:
I believe you can get rid of the `sm:` here for both the `z-*` and `absolute` classes, if I'm not mistaken it should have no effect, as it should be basically stating "for every screen with a width bigger than or equal to 640 pixels", which includes all the other breakpoints.

More on that on Tailwind's documentation → [TailwindCSS Breakpoint Docs](https://tailwindcss.com/docs/screens).

Reply:
I believe you addressed this already